rust/connlib/tunnel/src/client.rs

```diff
@@ -828,6 +827,8 @@ impl ClientState {
             .map(|(ip, _)| ip)
             .chain(iter::once(IPV4_RESOURCES.into()))
             .chain(iter::once(IPV6_RESOURCES.into()))
+            .chain(iter::once(DNS_SENTINELS_V4.into()))
+            .chain(iter::once(DNS_SENTINELS_V4.into()))
```

Comment:
```suggestion
            .chain(iter::once(DNS_SENTINELS_V6.into()))
```

Reply:
Probably didn't detect this because the sentinels ip is within the resources ips range

Reply:
Lol, same bug in the tests too :joy: